q2_assembly/spades/tests/test_spades.py

```diff
@@ -89,6 +95,10 @@ def generate_exp_calls(self, sample_ids, kind="paired"):
             exp_calls.append(call(f"sample{s}", fwd, rev, self.test_params_list, ANY))
         return exp_calls
 
+    def generate_exp_calls_coassembly(self, fwd, rev=None):
+        exp_calls = [call("all_samples_spades", fwd, rev, self.test_params_list, ANY)]
+        return exp_calls
+
```

Comment:
No need for the function here - you can just place the list in your test code directly.